distributed/bokeh/components.py

```diff
@@ -112,6 +112,8 @@ def __init__(self, n_rectangles=1000, clear_interval='20s', **kwargs):
             export.register_plot(self.root)
             self.root.add_tools(export)
 
+        self.root = row(self.root, name='task_stream', sizing_mode='scale_width')
```

Comment:
I think the desired pattern is to put the roots inside of ``bokeh.layout.row`` layouts with a responsive ``scale_width`` sizing mode and name, so that it's addressable in a template